### PR TITLE
Dildos4u - ReAdds Dildos

### PR DESCRIPTION
**Pull request by AliiTea**
They cannot fit in Ass unless you are an alien or have a bluespace ass.
# MakeHippieGreatAgain #BringBackDildos

honk

### CODE REVIEW

**Comment by trifornt-tinfornt** [comment]
No.

**Comment by calibraptor** [comment]
I approve.

Consider it motivation for R&D to research bluespace butts so they can shove dildos up their asses.

**Comment by ghost** [comment]
nope.

**Comment by myontasm** [comment]
Jesus Christ

**Comment by Lawligagger** [comment]
don't remove emag again pls

**Comment by AliiTea** [comment]
@Lawligagger I won't..

That was just a dumb joke.

**Comment by calibraptor** [comment]
We're descending into this pit of depravity and gloom, enveloping ourselves in the perversions of a weeaboo and stumbling, tumbling, falling into an endless cycle of audacious stupidity.

_But we're doing it in style motherfucker_

**Comment by ghost** [comment]
That's a no from me, sorry.

**Comment by AliiTea** [comment]
@oprayx73 if only a no from you meant anything...

**Comment by ghost** [comment]
that's a no from me sorry mate but im gonna have to say no

**Comment by Carbonhell** [comment]
Code is OK but It'd be preferred.if you didnt capitalize names, lowertext them please

**Comment by AliiTea** [comment]
@Carbonhell kept the parent and the shelf names unchanged as I don't feel they need to be changed.. that being said if they do need to be changed I will change them.

**Comment by doobiedoo23** [comment]
yea

**Comment by doobiedoo23** [comment]
don't capitalise shit tho it's inconsistent

**Comment by danhibiki** [comment]
die

**Comment by myontasm** [comment]
Still conflicted about this, emotionally.

**Comment by AliiTea** [comment]
Removed all caps @doobiedoo23 @Carbonhell

**Comment by doobiedoo23** [comment]
+/obj/item/dragon/sea
-  name = "sea dragon dildo"
-  desc = "A Sea Dragon Dildo, Why this is on a SpaceStation we will never know..."
-  icon_state = "seadragon"
-  w_class = 3
  +
  +/obj/item/dragon/canine
-  name = "canine dildo"
-  desc = "A Canine Dildo, Why this is on a SpaceStation we will never know..."
-  icon_state = "canine"
-  w_class = 3
-  throwforce = 7
  +
  +/obj/item/dragon/equine
-  name = "equine dildo"
-  desc = "A fucking Horse Cock!?  WHY!  Why do we have this shit on a Space Station..."
-  icon_state = "equine"
-  force = 11
-  w_class = 4
-  throwforce = 9

Descriptions are still capitalised

**Comment by myontasm** [comment]
The descriptions are pretty lame-o. Please rewrite them aswell.

**Comment by doobiedoo23** [comment]
desc = "A fucking Horse Cock!? WHY! Why do we have this shit on a Space Station..."

yeah this is just fucking awful tbh

**Comment by AliiTea** [comment]
Changed them again

**Comment by doobiedoo23** [comment]
Much better, still not crazy on the descriptions themselves but I'm asking too much

**Comment by Carbonhell** [comment]
descriptions must be capitalized, names don't,current code is ok

File: code/game/objects/items/dildo.dm [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	icon_state = null
+	force = 5
+	w_class = 1
+	throwforce = 5
+	hitsound = 'sound/items/squishy.ogg'
+	attack_verb = list("slapped")
+
+/obj/item/dragon/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is shoving [src.name] down \his throat! It looks like they're trying to commit suicide.</span>"))
+	return(BRUTELOSS)
```
Comment (Carbonhell) [inline code/game/objects/items/dildo.dm]: wait there's no space here

**Comment by Carbonhell** [comment]
actually lil bug, fix the note i made

File: code/game/objects/items/dildo.dm (hunk 1 above)
Comment (AliiTea) [inline code/game/objects/items/dildo.dm]: No space?

Comment (Carbonhell) [inline code/game/objects/items/dildo.dm]: it's currently return(BRUTELOSS), should be return (BRUTELOSS)

Comment (AliiTea) [inline code/game/objects/items/dildo.dm]: Oh.. fixed

**Comment by AliiTea** [comment]
reee
Nebyab has approved

**Comment by ghost** [comment]
Dunno if I could count Nebyab approval since due to their size, xeno and bluespace butts can fit them. He does explicitly state he's down with dildos provided they can't got up the bum

**Comment by AliiTea** [comment]
Bluespace butts need bluespace 5 and biotech 4   so you do have too put some effort in getting dildos into your butt

**Comment by ghost** [comment]
nebyab
I mean, the PR in its current state does allow dildos going up someone's bum which nebyab didn't want.

**Comment by AliiTea** [comment]
Until I get a reply I will keep the code how it is..

If Nebyab doesn't want them to work in any butt I will make it so.

**Comment by doobiedoo23** [comment]
why are you so obsessed with dildoes going up butts tho

**Comment by AliiTea** [comment]
I'm not.. the players will want it though so I am attempting to let them be able too.

**Comment by Pyko1** [comment]
As it is, dildos don't fit in normal butts, and it would take some serious butt-transplanting surgery to fit them in a players butt. To stop that it would need some snowflakey code, and then we would also consider do we need to butt-ban other fallos shaped objects, such as screwdrivers and baseball bats, and even then someone can make a book called as "HUGE AIDS INFESTED NIGGER COCK!!!!" and shove it in someones butt.

Ultimately, if someone wants to eRape someone he will pretty much always have means to do so as long as we have any chat functions. And forcibly showing dildos in someones butt should be dealt as any any other form of rape ERP.

Not that dildos really add anything to game besides immature humor, but if they get added they should be normal items with normal limits really.

**Comment by KayeArray** [comment]
yes

**Comment by Crystalwarrior** [comment]
No. No. No. No. No. No.
...
Yeah I'm 110% against this, I personally think that dildos are dumb gimmicky items that serve no other purpose than lulz.
Not to mention when dildos were actually live they had like 10 force and were everywhere in maint, making them arguably the best melee in the game availability and force-wise without much thought put into balance.
Also not to mention their insane goddamn embed chance.

**Comment by AliiTea** [comment]
These don't spawn anywhere currently, highest force is 11.

I have no idea how the new embed stuff works so I  don't know if they are crazy with embedding.

**Comment by ghost** [comment]
yes

**Comment by myontasm** [comment]
I'm still very much conflicted about this but yeah sure.

**Comment by Crystalwarrior** [comment]
If this gets committed I'll commit sudoku in dormitory toilets with a fireaxe.

**Comment by AliiTea** [comment]
Wooo almost there!

**Comment by Lawligagger** [comment]
http://puu.sh/ovBmU/cd34e01031.mp3

**Comment by AliiTea** [comment]
No fun.

**Comment by calibraptor** [comment]
@Crystalwarrior Rods do 10 brute.

**Pull request closed without merging**